dashboards/Storage-Summary.json

```diff
@@ -105,17 +306,45 @@
       "steppedLine": false,
       "targets": [
         {
-          "expr": "dss_sched_uptime",
+          "expr": "dss_obj_volumes_online",
           "format": "time_series",
           "intervalFactor": 1,
-          "legendFormat": "",
+          "legendFormat": "online",
           "refId": "A"
+        },
+        {
+          "expr": "dss_obj_volumes_pending",
+          "format": "time_series",
+          "intervalFactor": 1,
+          "legendFormat": "pending",
+          "refId": "B"
+        },
+        {
+          "expr": "dss_obj_volumes_removing",
+          "format": "time_series",
+          "intervalFactor": 1,
+          "legendFormat": "removing",
+          "refId": "C"
+        },
+        {
+          "expr": "dss_obj_volumes_stale",
+          "format": "time_series",
+          "intervalFactor": 1,
+          "legendFormat": "stale",
+          "refId": "D"
+        },
+        {
+          "expr": "dss_obj_volumes_recovery",
```

Comment:
Once https://github.com/mesosphere/dcos-storage/pull/470 lands we need to change this tot `dss_obj_devices_recovery`.

Reply:
Updated the dashboard with `devices` gauges as well.

<img width="1620" alt="Screen Shot 2019-05-10 at 10 39 45 AM" src="https://user-images.githubusercontent.com/324957/57514123-06119f80-7310-11e9-9152-4f228a40cbdc.png">

Reply:
OK - we can do the rename to `dss_obj_devices_recovery` now.

Reply:
Not sure I understand, we already show `dss_obj_devices_recovery`. Could you explain what need to be done here?

Reply:
Ah! I see - ok - I _think_ `recovery` is only applicable to `devices` (not volumes) and `stale` is only applicable to `volumes` (not devices). I'm not sure though and these conditions seem general enough that in future they may apply to both of these types of targets.

I'm happy with the dashboards as-is. Thanks, let's merge this.